commit be96bde117a3bef9576edc46b136d3716251f711
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Mon Nov 26 20:15:16 2018 +0000

    overlays: Update upstream overlay
    
    The vc4-kms-v3d overlay gained an extra fragment enabling the txp node,
    so rebuild the upstream overlay to match.
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/arch/arm/boot/dts/overlays/upstream-overlay.dts b/arch/arm/boot/dts/overlays/upstream-overlay.dts
index ff37a5013de4..b597f863c4a8 100644
--- a/arch/arm/boot/dts/overlays/upstream-overlay.dts
+++ b/arch/arm/boot/dts/overlays/upstream-overlay.dts
@@ -116,6 +116,12 @@
 		};
 	};
 	fragment@17 {
+		target = <&txp>;
+		__overlay__ {
+			status = "okay";
+		};
+	};
+	fragment@18 {
 		target = <&usb>;
 		#address-cells = <1>;
 		#size-cells = <1>;
@@ -130,21 +136,21 @@
 			status = "okay";
 		};
 	};
-	fragment@18 {
+	fragment@19 {
 		target = <&uart1>;
 		__overlay__ {
 			interrupt-parent = <&intc>;
 			interrupts = <0x1 0x1d>;
 		};
 	};
-	fragment@19 {
+	fragment@20 {
 		target = <&spi1>;
 		__overlay__ {
 			interrupt-parent = <&intc>;
 			interrupts = <0x1 0x1d>;
 		};
 	};
-	fragment@20 {
+	fragment@21 {
 		target = <&spi2>;
 		__overlay__ {
 			interrupt-parent = <&intc>;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1363_be96bde117a3bef9576edc46b136d3716251f711.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1363_be96bde117a3bef9576edc46b136d3716251f711.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
